mkt/ratings/tests/test_views.py

```diff
@@ -364,6 +364,45 @@ def test_already_rated_version(self):
         assert data['user']['can_rate']
         assert not data['user']['has_rated']
 
+    def test_no_lang_filter(self):
+        Review.objects.create(addon=self.app, user=self.user, body='yes',
+                              lang='en')
+        other_user = UserProfile.objects.exclude(pk=self.user.pk)[0]
+        Review.objects.create(addon=self.app, user=other_user, body='yes',
+                              lang='pt')
+        res, data = self._get_url(self.list_url, app=self.app.app_slug,
+                                  client=self.anon, lang='pt')
+        eq_(res.status_code, 200)
+        eq_(len(data['objects']), 2)
+        eq_(data['info']['total_reviews'], 2)
+
+    def test_lang_filter(self):
+        Review.objects.create(addon=self.app, user=self.user, body='yes',
+                              lang='en')
+        other_user = UserProfile.objects.exclude(pk=self.user.pk)[0]
+        Review.objects.create(addon=self.app, user=other_user, body='yes',
+                              lang='pt')
+        res, data = self._get_url(self.list_url, app=self.app.app_slug,
+                                  client=self.anon, lang='pt',
+                                  match_lang='1')
+        eq_(res.status_code, 200)
+        eq_(len(data['objects']), 1)
+        eq_(data['info']['total_reviews'], 2)
+        eq_(data['objects'][0]['lang'], 'pt')
+
+    def test_lang_filter_inverse(self):
+        Review.objects.create(addon=self.app, user=self.user, body='yes',
+                              lang='en')
+        other_user = UserProfile.objects.exclude(pk=self.user.pk)[0]
+        Review.objects.create(addon=self.app, user=other_user, body='yes',
+                              lang='pt')
+        res, data = self._get_url(self.list_url, app=self.app.app_slug,
+                                  client=self.anon, lang='pt',
+                                  match_lang='0')
```

Comment:
This test should likely be removed along with the documentation saying that `match_lang=0` gives ratings in other languages.